### DISCUSSION TITLE
Option for obstacles and monsters in Sub editor test

### DISCUSSION THREAD

**Discussion by Cogs16** [FakeFishGames/Barotrauma#5835] [Ideas and suggestions]
Add an option to have some random chunks of terrain placed around the sub in the editor's test mode. This would enable crash, and collisions tests with the main sub. Additionally, being able to say have auto spawning monsters or a selectable # of bots would also be nice QOL features., even if they can be done with console.

**Comment by Ek-Videogames** [comment]
Basically `quickstart` command but better. 👍
